### ISSUE TITLE
3D Pack support

### ISSUE THREAD

**Issue by yosun** [fofr/cog-comfyui#109]
https://github.com/MrForExample/ComfyUI-3D-Pack 

?

**Comment by yosun** [comment]
yeah comfy can get super wild - also video to 3d nodes too. iirc replicate has a 2 image limit as input. comfy AST supports N multimodal inputs - so could kind of be a way to bypass that limitation

**Comment by fofr** [comment]
You can pass in a zip of files

**Comment by dscalvin97** [comment]
Not sure if this is within the scope of this request. If adding support is not a viable option, would it be possible to create some kind of guide to help those setting up forks to add support correctly?
